Services/data/devicemodels/cab-01-protobuf.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
indentation: 4 spaces

Reply:
This is resolved. Please validate and update.